### PR TITLE
Update contact

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: content/docs/faq.md [hunk 1]
```diff
@@ -31,8 +31,10 @@ If you encounter a problem while using packit or Packit Service, please open an
 In case of any other questions, feel free to contact
 [us](https://github.com/orgs/packit/teams/the-packit-team):
 
-- **#packit** on [Libera.Chat](https://libera.chat/)
+- [**#packit:fedora.im**](https://chat.fedoraproject.org/#/room/#packit:fedora.im) (Element/Matrix)
+- [**#packit:libera.chat**](https://libera.chat) (IRC)
 - **hello@packit.dev**
+- [**@packit@fosstodon.org**](https://fosstodon.org/@packit) (Mastodon)
```
Comment (lachmanfrantisek) [inline content/docs/faq.md:37]: Do we want to maintain the contacts in two places?

What about linking the one on the main page from here?

File: content/_index.md [hunk 2]
```diff
@@ -65,15 +65,8 @@ and create [Bodhi](https://bodhi.fedoraproject.org) updates! [Learn more.](/docs
...
-   [us](https://github.com/orgs/packit/people):
-
-#### Contact
-
-- **#packit:fedora.im** on [Matrix](https://matrix.org/)
-- **#packit** on [Libera.Chat](https://libera.chat/)
-- **hello@packit.dev**
-- **@packit** on {{< rawhtml >}}<a rel="me" href="https://fosstodon.org/@packit">Fosstodon</a>{{< /rawhtml >}}
+4. _We are here to help_ - In case of questions, feel free to [contact us]({{< ref
+   "faq#how-can-i-contact-you" >}}).
```
Comment (lachmanfrantisek) [inline content/_index.md:69]: Sorry. I meant the opposite way. :speak_no_evil: 

(The contact header on the homepage is linked from multiple places in the service so I would preserve that one.)

File: content/docs/faq.md (hunk 1 above)
Comment (jpopelka) [reply content/docs/faq.md]: I agree with not having it in two places, but I did it the other way round, i.e. left the contacts in FAQ and linked them from the main page. The main page is the entry point and I don't think the info about how to contact us is something someone who reads about Packit for the first time needs to know to decide whether Packit is a good fit for them.

File: content/_index.md (hunk 2 above)
Comment (jpopelka) [reply content/_index.md:69]: I know :-) I can change the "multiple places" as well.
See my above comment for reasoning.

But I'm not against leaving it the original way if you insist.

Comment (jpopelka) [reply content/_index.md]: Reverted to the original layout, the `#contact` is used also in code and presentations.

**Review by lbarcziova — APPROVED** [review]
thanks!

**Review by lachmanfrantisek — APPROVED** [review]

**Comment by mfocko** [comment]
@jpopelka Zuul seems to be down, https://softwarefactory-project.io/zuul/t/local/status, you probably want to merge manually; or we can set up a merge queue :upside_down_face:

**Review by mfocko — APPROVED** [review]

**Pull request merged**